lib/modules/platform/bitbucket/index.ts

```diff
@@ -230,6 +231,27 @@ export async function initRepo({
 
     config.defaultBranch = mainBranch;
 
+    if (bbMendAppDashboardStatus) {
```

Comment:
I don't think this be available here, if set in repo config? The option should be self-hosted